CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# [6.12.0](https://github.com/algolia/react-instantsearch/compare/v6.11.0...v6.12.0) (2021-07-06)
+
+
+### Bug Fixes
+
+* **DynamicWidgets:** read from facets.order ([5835c19](https://github.com/algolia/react-instantsearch/commit/5835c19))
+* **HitsPerPage:** Adds id prop to HitsPerPage, Select components ([#3072](https://github.com/algolia/react-instantsearch/issues/3072)) ([bc75d75](https://github.com/algolia/react-instantsearch/commit/bc75d75))
+* **MenuSelect:** Adds id prop to MenuSelect ([#3073](https://github.com/algolia/react-instantsearch/issues/3073)) ([fddaaef](https://github.com/algolia/react-instantsearch/commit/fddaaef))
+* **SearchBox:** Adds inputId prop to SearchBox ([#3074](https://github.com/algolia/react-instantsearch/issues/3074)) ([a05f6a4](https://github.com/algolia/react-instantsearch/commit/a05f6a4))
+* **SortBy:** Adds `id` prop to `SortBy`, `Select` components ([#3068](https://github.com/algolia/react-instantsearch/issues/3068)) ([1f2797f](https://github.com/algolia/react-instantsearch/commit/1f2797f))
+
+
+### Features
+
+* **DynamicWidgets:** add fallbackComponent ([#3066](https://github.com/algolia/react-instantsearch/issues/3066)) ([1217694](https://github.com/algolia/react-instantsearch/commit/1217694))
```

Comment:
should we leave it here or remove it since it was not shipped until now? cc @Haroenv

Reply:
```suggestion
```